api/views.py

```diff
@@ -49,6 +51,40 @@ class TimerViewSet(viewsets.ModelViewSet):
     serializer_class = serializers.TimerSerializer
     filterset_class = filters.TimerFilter
 
+    def __timer_operation(self, pk, func):
+        try:
+            timer = models.Timer.objects.get(pk=pk)
+            return func(timer)
+        except models.Timer.DoesNotExist:
+            return Response(
+                {"detail": "timer does not exist"},
```

Comment:
We don't use `detail` anywhere else in the API. We do use `non_field_errors` but that doesn't seem to fit here. Lets just return `404` with no body.

Reply:
Ok sounds good. I tried to read the docs for the Django REST framework, and these (including the PATCH-change) seemed to match the behavior from there the best - you can see this when trying to obtain, for example, a timer that does not exist with: http://127.0.0.1:8000/api/timers/1232/

But only a 404 is also fine and logic.

Reply:
Oh wait -- you're right. I didn't realize that. Lets stick with what DRF does for this, and use the same simple detail of `{"detail":"Not found."}`. Thanks for catching.

Reply:
Yeah, no problem. That is why a friendly push and pull can be helpful, right? "Not found" it is!

I will have a look, if I can leverage the code of Django REST Framework somehow for generating this message. Seems like the "nicest" solution but not sure if that is possible somehow. I can namely see on my end that the response messages from the REST API are also localized to German somehow. Probably would be great to just leverage what is there already.

Reply:
Yeah part of why I was thinking to get rid of the detail entirely was one less thing to translate. I'm not sure if DRF does that though... Baby Buddy's language is based on a custom user setting so most certainly not. Regardless, anything we can inherit from DRP is a plus!

Reply:
(I think this is resolved, I will write a short digest below)